src/screens/ChannelListScreen/elements/Search.js

```diff
@@ -86,7 +87,7 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[400],
     fontSize: 14,
     alignSelf: 'center',
-    color: COLORS.gray9
+    color: colors.gray1
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color constant has been updated. Make sure this new color constant is defined and correctly represents the intended color.
```diff
-    color: COLORS.gray9
+    color: colors.gray1
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->